domains/cyberwizard.json

```
@@ -0,0 +1,10 @@
+{
+  "domain": "cyberwizard.is-a.dev",
+  "owner": {
```

```suggestion
  "owner": {
  "username": "Programmer-Dev-King",
```
This field is **required**.